module/dkg/broker.go

```diff
@@ -89,12 +97,20 @@ func (b *Broker) PrivateSend(dest int, data []byte) {
 func (b *Broker) Broadcast(data []byte) {
 	bcastMsg, err := b.prepareBroadcastMessage(data)
 	if err != nil {
-		b.log.Error().Err(err).Msg("could not create broadcast message")
+		b.log.Error().Err(err).Msg("failed to create broadcast message")
 	}
```

Comment:
```suggestion
		b.log.Fatal().Err(err).Msg("failed to create broadcast message")
	}
```

Are there any circumstances where we would expect this error?

Reply:
Only if it fails to sign the message.. which is probably a fatal error.